src/client/app/modules/routes/edit-routes.jade

```diff
@@ -12,6 +12,28 @@ form(name="editForm")
     label Persistent data
     p {{ vm.formInfo.data.length === 0 ? 'No data' : vm.formInfo.data | json }}
 
+    .alert.alert-info
+    p Sort Type: {{ vm.sortType }}
+    p Sort Reverse: {{ vm.sortReverse }}
+    p Search Query: {{ vm.searchData }}
+    form
+      .form-group
+        .input-group
+          .input-group-addon
+            i.fa.fa-search
+          input.form-control(type='text', placeholder='Search', ng-model='vm.searchData')
```

Comment:
this style of cascading classes is not intuitive or needed if it'll only apply to one element. (also, didn't know it was possible -- really neat)